src/components/Image.js

```diff
@@ -11,9 +12,25 @@ const styles = StyleSheet.create({
     padding: 0,
     marginBottom: 5,
     backgroundColor: 'grey'
+  },
+  alt: {
+    fontSize: 8,
+    lineHeight: 1.3,
+    fontFamily: fontFamilies.sansSerifRegular,
+    marginBottom: 5
   }
 })
 
-export default ({ src }) => {
+export const Alt = ({alt, children}) => (
```

Comment:
We are not doing anything with the `alt` prop in here right?
Maybe having the `Alt` in a different component it's not necessary